source/content/pantheon-yml.md

```diff
@@ -171,7 +171,7 @@ Users of Drupal 6 sites should consider [upgrading to Drupal 7](/drupal-updates#
 
 #### Considerations - Drupal 9
```

Comment:
@stovak , multiple references to d9 in doc https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/pantheon-yml